packages/react-keytips/stories/Overflow/View/keytipsMap.ts

```diff
@@ -0,0 +1,54 @@
+import { ExecuteKeytipEventHandler } from '@fluentui-contrib/react-keytips';
+
+export const onExecute: ExecuteKeytipEventHandler = (_, { targetElement }) => {
+  if (targetElement) {
+    console.info(targetElement);
+    targetElement.focus();
+    targetElement.click();
+  }
+};
+
```

Comment:
Maybe it's better to move this method to some shared file, because it's duplicated in `Home` and `Help` folders.